src/components/tables/Connectors/Rows.tsx

```diff
@@ -0,0 +1,89 @@
+import { Box, Button, TableCell, TableRow } from '@mui/material';
+import { routeDetails } from 'app/Authenticated';
+import ExternalLink from 'components/shared/ExternalLink';
+import { Connector } from 'components/tables/Connectors';
+import { FormattedDate, FormattedMessage } from 'react-intl';
+import { useNavigate } from 'react-router';
+
+interface Props {
+    data: Connector[];
+}
+const columnStyling = {
+    maxWidth: '20%',
+    textOverflow: 'ellipsis',
+    width: '20%',
+};
+
+function Rows({ data }: Props) {
+    const navigate = useNavigate();
+
+    return (
+        <>
+            {data.map((row) => (
+                <TableRow key={`Connector-${row.id}`}>
+                    <TableCell style={columnStyling}>
+                        {row.image_name}
+                    </TableCell>
+                    <TableCell style={columnStyling}>{row.detail}</TableCell>
+                    <TableCell style={columnStyling}>
+                        {row.connector_tags[0].protocol}
+                    </TableCell>
+                    <TableCell style={columnStyling}>
+                        <FormattedDate
+                            day="numeric"
+                            month="long"
+                            year="numeric"
+                            value={row.updated_at}
+                        />
+                    </TableCell>
+                    <TableCell style={columnStyling}>
+                        {row.connector_tags[0].documentation_url ? (
+                            <ExternalLink
+                                link={row.connector_tags[0].documentation_url}
+                            >
+                                Docs
+                            </ExternalLink>
+                        ) : (
+                            <>N/A</>
```

Comment:
I think this would make a good component. We now have two spots where links to docs are put. Also - this will be an important UX for users.